addon/wolfcrypt/h/evp.h

```diff
@@ -21,7 +21,11 @@
 #include <napi.h>
 #include <stdio.h>
 #include <cstring>
+#ifndef WOLFSSL_USER_SETTINGS
 #include "wolfssl/options.h"
+#else
+#include "../user_settings.h"
+#endif
```

Comment:
`options.h`/`user_settings.h` inclusion can be controlled by defining `WOLFSSL_USER_SETTINGS` and/or `WOLFSSL_USE_OPTIONS_H` before including `wolfssl/wolfcrypt/settings.h`, and include `settings.h` before any other libwolfssl headers.  My personal preference is to include `wolfssl/wolfcrypt/types.h` first, because it includes `wolfssl/wolfcrypt/settings.h` first, then gets all the basic types in place.